spec/index.bs

```diff
@@ -7441,7 +7441,7 @@ dictionary GPURenderPassLayout: GPUObjectDescriptorBase {
 
             Issue the following steps on the [=Device timeline=] of |this|.{{GPUObjectBase/[[device]]}}:
             <div class=device-timeline>
-                1. If |size| is missing, set |size| to |buffer|.{{GPUBuffer/[[size]]}} - |offset|.
+                1. If |size| is missing, set |size| to max(0, |buffer|.{{GPUBuffer/[[size]]}} - |offset|).
```

Comment:
Seems we're missing a validation rule that `offset < buffer size`? I think it makes sense to validate that.

Reply:
The combination of this and `offset + size ≤ buffer.[[size]]` validate that the offset is inside the buffer size. Unless we want to add `offset <= buffer.size` everywhere?

Reply:
Ah I see adding the max() makes it so we don't need that rule explicitly. Never mind then!